Libraries/CommonFunctions.psm1

```diff
@@ -116,20 +69,47 @@ Function Select-TestCases($TestXMLs, $TestCategory, $TestArea, $TestNames, $Test
     $AllLisaTests =  [System.Collections.ArrayList]@()
     $WildCards = @('^','.','[',']','?','+','*')
     $ExcludedTestsCount = 0
-
-    # if the expected filter parameter is 'ALL' or empty ("ALL" -imatch "" return $True), the actually filter used in this function will be '*', except for '$ExcludedTests'
-    if ("All" -imatch $TestCategory)   { $TestCategory = "*" }
-    if ("All" -imatch $TestArea)       { $TestArea = "*" }
-    if ("All" -imatch $TestNames)      { $TestNames = "*" }
-    if ("All" -imatch  $TestTag)       { $TestTag = "*" }
-    if ("All" -imatch  $TestPriority)  { $TestPriority = "*" }
-    if ("All" -imatch  $TestSetup)     { $TestSetup = "*" }
-
-    $testCategoryArray = @($TestCategory.Trim(', ').Split(',').Trim())
-    $testAreaArray = @($TestArea.Trim(', ').Split(',').Trim())
-    $testNamesArray = @($TestNames.Trim(', ').Split(',').Trim())
-    $testSetupTypeArray = @($TestSetup.Trim(', ').Split(',').Trim())
-    $excludedTestsArray = @($ExcludeTests.Trim(', ').Split(',').Trim())
+    $testCategoryArray = $testAreaArray = $testNamesArray = $testTagArray = $testPriorityArray = $testSetupTypeArray = $excludedTestsArray = @()
+    # if the expected filter parameter is 'All' (case insensitive) or empty or $null, the actually filter used in this function will be '*', except for '$ExcludedTests'
```

Comment:
if the expected filter parameter is 'All' (case insensitive) or empty or $null ---> if the expected filter parameter is 'All' (case insensitive), empty or $null

Reply:
actually -> actual